src/Core/MerchandiseReturn/Configuration/MerchandiseReturnOptionsConfiguration.php

```diff
@@ -26,36 +26,31 @@
 
 namespace PrestaShop\PrestaShop\Core\MerchandiseReturn\Configuration;
 
-use PrestaShop\PrestaShop\Core\Configuration\DataConfigurationInterface;
-use PrestaShop\PrestaShop\Core\ConfigurationInterface;
+use PrestaShop\PrestaShop\Core\Configuration\AbstractMultistoreConfiguration;
+use Symfony\Component\OptionsResolver\OptionsResolver;
 
 /**
  * Provides data configuration for merchandise returns options form
  */
-final class MerchandiseReturnOptionsConfiguration implements DataConfigurationInterface
+class MerchandiseReturnOptionsConfiguration extends AbstractMultistoreConfiguration
 {
-    /**
-     * @var ConfigurationInterface
-     */
-    private $configuration;
-
-    /**
-     * @param ConfigurationInterface $configuration
-     */
-    public function __construct(ConfigurationInterface $configuration)
-    {
-        $this->configuration = $configuration;
-    }
+    private const CONFIGURATION_FIELDS = [
+        'enable_order_return',
+        'order_return_period_in_days',
```

Comment:
Why don't you use const in MerchandiseReturnOptionsType, to define configuration fields?